### PR TITLE
Feature/aes master keys

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

File: src/main/java/com/kryptnostic/directory/v1/http/DirectoryApi.java [hunk 1]
```diff
@@ -115,6 +117,19 @@ private PARAM() {}
     @PUT( CONTROLLER + PRIVATE_KEY )
     BasicResponse<String> setPrivateKey( @Body BlockCiphertext encryptedPrivateKey );
 
+    @Deprecated
+    @Timed
+    @GET( CONTROLLER + MASTER_KEY )
+    byte[] getMasterKey();
```
Comment (makosblade) [inline src/main/java/com/kryptnostic/directory/v1/http/DirectoryApi.java]: What's the plan for this?

Comment (makosblade) [inline src/main/java/com/kryptnostic/directory/v1/http/DirectoryApi.java]: Also this should probably go in the keystorage service, no?

Comment (makosblade) [inline src/main/java/com/kryptnostic/directory/v1/http/DirectoryApi.java]: Nvm on plan, missed an email

Comment (geekbeast) [inline src/main/java/com/kryptnostic/directory/v1/http/DirectoryApi.java]: This change is also going to require a front-end migration step, which is why I had to make changes the User class in the first place.

**Pull request merged**